src/System/Relay/createRelaySSREnvironment.ts

```diff
@@ -108,7 +108,7 @@ export function createRelaySSREnvironment(config: Config = {}) {
     cacheMiddleware({
       size: Number(getENV("NETWORK_CACHE_SIZE")) ?? 2000, // max 2000 requests
       ttl: Number(getENV("NETWORK_CACHE_TTL")) ?? 3600000, // 1 hour
-      clearOnMutation: true,
+      clearOnMutation: false,
```

Comment:
This setting can probably be left out altogether from the config, but explicit `false` is fine too.

Reply:
Yah lets keep it visible (and in the git diff)